src/main/webapp/student-ambassador.html

```diff
@@ -252,12 +240,18 @@ <h5 class="mb-0">
 <script src="https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/14.0.3/nouislider.min.js"></script>
 <!-- Argon JS -->
 <script src="assets/js/argon.js?v=1.0.1"></script>
-<script src="assets/js/global-custom-js.js"></script>
 <!-- Script for static navbar and footer -->
 <script src="assets/js/navigation.js"></script>
+<script src="assets/js/function.js"></script>
 <script>
     $(function () {
         loadNavAndFooter('assets/content/static')  //relative path to content directory
     })
 </script>
+
+<script>
+    $(function () {
+        faq()
```

Comment:
add semicolon;

Reply:
Use the same ready function. (move the function call to below `loadNavAndFooter('assets/content/static');`)